packages/profile-sync-controller/src/controllers/user-storage/encryption/encryption.ts

```diff
@@ -156,10 +187,11 @@ class EncryptorDecryptor {
     return gcm(key, nonce).decrypt(ciphertext);
   }
 
-  #getOrGenerateScryptKey(
+  async #getOrGenerateScryptKey(
     password: string,
     o: EncryptedPayload['o'],
     salt?: Uint8Array,
+    nativeScryptCrypto?: NativeScrypt,
```

Comment:
nice setting this to async (and the prop drilling lol).

We'll figure out a nice way of drilling down props or a context or something in a diff PR 😄